appveyor.yml

```diff
@@ -0,0 +1,20 @@
+version: 1.0.{build}
+build_cloud: DESKTOP-3EK2821
+image: Windows
+environment:
+  nuget_user: zuha@dslbd.onmicrosoft.com
+  nuget_password: Sakeen&Sakeen1
```

Comment:
@dslzuha is this intentionally?

Reply:
@Rud5G Thanks for informing me. This is an accident